src/libraries/System.IO.FileSystem.AccessControl/src/System.IO.FileSystem.AccessControl.csproj

```diff
@@ -5,6 +5,7 @@
     <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetsWindows)' != 'true'">SR.PlatformNotSupported_AccessControl</GeneratePlatformNotSupportedAssemblyMessage>
     <Configurations>net461-Windows_NT-Debug;net461-Windows_NT-Release;$(NetCoreAppCurrent)-Windows_NT-Debug;$(NetCoreAppCurrent)-Windows_NT-Release;$(NetFrameworkCurrent)-Windows_NT-Debug;$(NetFrameworkCurrent)-Windows_NT-Release;netstandard2.0-Debug;netstandard2.0-Release;netstandard2.0-Windows_NT-Debug;netstandard2.0-Windows_NT-Release</Configurations>
     <Nullable>annotations</Nullable>
```

Comment:
```suggestion
```

Reply:
The fact that this had `<Nullable>annotations</Nullable>` suggests there were files this was pulling in that were `#nullable enable`'d as part of annotating another library.  Can those files have their `#nullable enable`s removed now that this library is enabled as well?

Reply:
I thought it was for adding Nullable attributes reference for lower environments, no? CC @eiriktsarpalis

Reply:
Never mind I misunderstood 😝, will remove `<Nullable>annotations</Nullable>`, but not sure about #nullable enable as they still might been used somewhere ...

Reply:
They are used in Microsoft.IO.Redist, so will leave them for now